Source/Atj/data/mvp_scenario.json

```diff
@@ -262,12 +283,24 @@
                     "behavior": "move_to",
                     "target": "printer_1"
                 },
+                {
+                    "type": "behavior",
+                    "sync_time": "10.0",
+                    "behavior": "pick_up",
+                    "target": "big_meeting_agenda_1"
+                },
                 {
                     "type": "behavior",
                     "sync_time": "20.0",
                     "behavior": "move_to",
                     "target": "bulletin_board_1"
                 },
+                {
+                    "type": "behavior",
+                    "sync_time": "1.0",
+                    "behavior": "put_down",
+                    "target": "big_meeting_agenda_1"
```

Comment:
This part was a bit of a trap. The `target` of a ` put_down` action is actually an `object` rather than an `item`. The idea is that the NPC will put "whatever item they may or not be holding" into a particular `object`'s `slot`. If you didn't specify the object, we would have to guess which object to slot the item into, which could lead to finicky behavior.

On the flip side, you've got the `pick_up` behavior implemented correctly, as the `target` should in fact be an item. The idea here is that there's no ambiguity about what should be picked up if you specify the item name.

You can think of this as "what should I pick up" and "where should I put this down"

Reply:
This was an easy enough fix. I was able to look at the 'move_to' locations for every Routine and just copy that forward as the target of the action. I kept 'place item' as the same, so that should still be fine. All 'put_down' targets have been appropriately modified